documentation/nginx_service.md

```diff
@@ -0,0 +1,41 @@
+# nginx_service
+
+[Back to resource list](../README.md#resources)
+
+## Actions
+
+- `:start`
+- `:stop`
+- `:restart`
+- `:reload`
+- `:enable`
+- `:disable`
+
+## Properties
+
+| Name                   | Type          | Default                          | Description                                                         | Allowed Values      |
```

Comment:
Is there any reason why we need the `Allowed Values` column if we're not using it?